x-pack/plugins/ml/public/application/components/job_selector/_job_selector.scss

```diff
@@ -26,54 +26,54 @@
 }
 
 .mlJobSelector__ganttBarRunning {
-  background-image:-webkit-gradient(linear,
-    0 100%, 100% 0,
-    color-stop(0.25, rgba(255, 255, 255, 0.15)),
-    color-stop(0.25, transparent),
-    color-stop(0.5, transparent),
-    color-stop(0.5, rgba(255, 255, 255, 0.15)),
-    color-stop(0.75, rgba(255, 255, 255, 0.15)),
-    color-stop(0.75, transparent),
-    to(transparent));
-  background-image:-webkit-linear-gradient(45deg,
-    rgba(255, 255, 255, 0.15) 25%,
-    transparent 25%, transparent 50%,
-    rgba(255, 255, 255, 0.15) 50%,
-    rgba(255, 255, 255, 0.15) 75%,
-    transparent 75%,
-    transparent);
-  background-image:-moz-linear-gradient(45deg,
-    rgba(255, 255, 255, 0.15) 25%,
-    transparent 25%,
-    transparent 50%,
-    rgba(255, 255, 255, 0.15) 50%,
-    rgba(255, 255, 255, 0.15) 75%,
-    transparent 75%,
-    transparent);
-  background-image:-o-linear-gradient(45deg,
-    rgba(255, 255, 255, 0.15) 25%,
-    transparent 25%,
-    transparent 50%,
-    rgba(255, 255, 255, 0.15) 50%,
-    rgba(255, 255, 255, 0.15) 75%,
-    transparent 75%,
-    transparent);
-  background-image:linear-gradient(45deg,
-    rgba(255, 255, 255, 0.15) 25%,
-    transparent 25%,
-    transparent 50%,
-    rgba(255, 255, 255, 0.15) 50%,
-    rgba(255, 255, 255, 0.15) 75%,
-    transparent 75%,
-    transparent);
-  -webkit-background-size:40px 40px;
-  -moz-background-size:40px 40px;
-  -o-background-size:40px 40px;
-  background-size:40px 40px;
+  background-image: -webkit-gradient(linear,
+      0 100%, 100% 0,
+      color-stop(0.25, rgba(255, 255, 255, 0.15)),
+      color-stop(0.25, transparent),
+      color-stop(0.5, transparent),
+      color-stop(0.5, rgba(255, 255, 255, 0.15)),
+      color-stop(0.75, rgba(255, 255, 255, 0.15)),
+      color-stop(0.75, transparent),
+      to(transparent));
+  background-image: -webkit-linear-gradient(45deg,
+      rgba(255, 255, 255, 0.15) 25%,
+      transparent 25%, transparent 50%,
+      rgba(255, 255, 255, 0.15) 50%,
+      rgba(255, 255, 255, 0.15) 75%,
+      transparent 75%,
+      transparent);
+  background-image: -moz-linear-gradient(45deg,
+      rgba(255, 255, 255, 0.15) 25%,
+      transparent 25%,
+      transparent 50%,
+      rgba(255, 255, 255, 0.15) 50%,
+      rgba(255, 255, 255, 0.15) 75%,
+      transparent 75%,
+      transparent);
+  background-image: -o-linear-gradient(45deg,
+      rgba(255, 255, 255, 0.15) 25%,
+      transparent 25%,
+      transparent 50%,
+      rgba(255, 255, 255, 0.15) 50%,
+      rgba(255, 255, 255, 0.15) 75%,
+      transparent 75%,
+      transparent);
+  background-image: linear-gradient(45deg,
+      rgba(255, 255, 255, 0.15) 25%,
+      transparent 25%,
+      transparent 50%,
+      rgba(255, 255, 255, 0.15) 50%,
+      rgba(255, 255, 255, 0.15) 75%,
+      transparent 75%,
+      transparent);
+  -webkit-background-size: 40px 40px;
+  -moz-background-size: 40px 40px;
+  -o-background-size: 40px 40px;
+  background-size: 40px 40px;
```

Comment:
postCSS will do all the vendor prefixing. You can replace the pixel values here with EUI vars.